### ISSUE TITLE
Pasting an image URL from 2e AONprd website as layer in Tokenizer does not work.

### ISSUE THREAD

**Issue by chaoticnote** [MrPrimate/tokenizer#107]
Program: Foundry VTT, version 10 build 291
Game System: Pathfinder Second Edition (Official), v4.4.2
Tokenizer version 3.9.5

Steps Taken:
1. Open a World with the PF2e system.
2. Open an Actor's sheet from any of the PF2e compendium.
3. Click on Avatar section, which opens Tokenizer for that Actor's sheet.
4. Disable the "Modify Token Only?" option.
5. Attempt to Add Layer using a URL of an image from [https://2e.aonprd.com/](https://2e.aonprd.com/) on Avatar side.

In my case, I've tried modifying the Wolf sheet's avatar by uploading an image URL of the Wolf picture from the 2e Archives of Nethys website. However, it does not upload the image. Doing the same on the Token side of the Tokenizer has no results either. Unlocking the compendium it's in or importing the compendium to the Actors tab also does not fix the issue. However, when I upload any other image URL from, say, a wolf image from Wikipedia, that works just fine. I'd like to know why specifically uploading image URLs from the 2e AONprd website does not work and if this could be fixed somehow.

**Comment by chaoticnote** [comment]
Just tried with a Deviant Art image URL of a Shadow Drake drawn for 1e. Also does not work.

**Comment by chaoticnote** [comment]
Opened Console to see what's going on.

> commons.js:1703 Error: You are accessing the NPCPF2e#data object which is no longer used. Since V10 the Document class and its contained DataModel are merged into a combined data structure. You should now reference keys which were previously contained within the data object directly.
    at logCompatibilityWarning (commons.js:1692:19)
    at NPCPF2e._logV10CompatibilityWarning (commons.js:6642:14)
    at get data [as data] (commons.js:6542:24)
    at tokenizeActor (hooks.js:104:17)
    at tokenizeDoc (hooks.js:142:5)
    at HTMLImageElement.<anonymous> (hooks.js:283:15)
    at HTMLImageElement.dispatch (jquery.min.js:2:43064)
    at y.handle (jquery.min.js:2:41048)
logCompatibilityWarning @ commons.js:1703
_logV10CompatibilityWarning @ commons.js:6642
get data @ commons.js:6542
tokenizeActor @ hooks.js:104
tokenizeDoc @ hooks.js:142
(anonymous) @ hooks.js:283
dispatch @ jquery.min.js:2
y.handle @ jquery.min.js:2
foundry.js:5099 Foundry VTT | Rendering Tokenizer
3libWrapper-wrapper.js:669 Uncaught TypeError: Cannot read properties of undefined (reading 'querySelectorAll')
[Detected 2 packages: damage-log, lib-wrapper]
    at ChatLogPF2e._onUpdateTimestamps (damage-log.js:306:36)
    at 🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:616:16)
    at 🎁ChatLog.prototype.updateTimestamps#0 (libWrapper-wrapper.js:189:20)
_onUpdateTimestamps @ damage-log.js:306
🎁call_wrapper @ libWrapper-wrapper.js:616
🎁ChatLog.prototype.updateTimestamps#0 @ libWrapper-wrapper.js:189
foundry.js:5099 Foundry VTT | Rendering Dialog
libWrapper-wrapper.js:669 Uncaught TypeError: Cannot read properties of undefined (reading 'querySelectorAll')
[Detected 2 packages: damage-log, lib-wrapper]
    at ChatLogPF2e._onUpdateTimestamps (damage-log.js:306:36)
    at 🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:616:16)
    at 🎁ChatLog.prototype.updateTimestamps#0 (libWrapper-wrapper.js:189:20)
_onUpdateTimestamps @ damage-log.js:306
🎁call_wrapper @ libWrapper-wrapper.js:616
🎁ChatLog.prototype.updateTimestamps#0 @ libWrapper-wrapper.js:189
game:1 Access to image at 'https://2e.aonprd.com/Images/Monsters/Drake_ShadowDrake.png?1670775195258' from origin 'http://localhost:30000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
2e.aonprd.com/Images/Monsters/Drake_ShadowDrake.png?1670775195258:1          GET https://2e.aonprd.com/Images/Monsters/Drake_ShadowDrake.png?1670775195258 net::ERR_FAILED 302
Image (async)
(anonymous) @ Utils.js:160
download @ Utils.js:130
await in download (async)
callback @ Tokenizer.js:400
submit @ foundry.js:56403
_onClickButton @ foundry.js:56332
dispatch @ jquery.min.js:2
y.handle @ jquery.min.js:2
logger.js:63 Tokenizer | ERR > Download listener error Event {isTrusted: true, type: 'error', target: img, currentTarget: img, eventPhase: 2, …}
log @ logger.js:63
error @ logger.js:86
img.onerror @ Utils.js:144
error (async)
(anonymous) @ Utils.js:143
download @ Utils.js:130
await in download (async)
callback @ Tokenizer.js:400
submit @ foundry.js:56403
_onClickButton @ foundry.js:56332
dispatch @ jquery.min.js:2
y.handle @ jquery.min.js:2
logger.js:63 Tokenizer | ERR > Error fetching image Event {isTrusted: true, type: 'error', target: img, currentTarget: img, eventPhase: 2, …}
log @ logger.js:63
error @ logger.js:86
(anonymous) @ Tokenizer.js:403
Promise.catch (async)
callback @ Tokenizer.js:402
submit @ foundry.js:56403
_onClickButton @ foundry.js:56332
dispatch @ jquery.min.js:2
y.handle @ jquery.min.js:2
Tokenizer.js:404 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'error')
[Detected 1 package: vtta-tokenizer]
    at Tokenizer.js:404:39

**Comment by MrPrimate** [comment]
This is a dupicate of https://github.com/MrPrimate/tokenizer/issues/26 please try copy-pasting the image instead. (You can use ctrl-v to paste the image into tokenizer).
